mailer/index.js

```diff
@@ -43,11 +43,17 @@ const EMAIL_CONTENT = {
     contentTop:
       'Ballers just made you an offer for property [Property name]. Valid till [Expiry date]. Check your dashboard for more details.',
   },
-  OFFER_RESPONSE: {
+  OFFER_RESPONSE_VENDOR: {
     subject: 'Offer Accepted',
     contentTop:
       'Your offer on [Property name] has been accepted. Check your dashboard for more details.',
   },
+  OFFER_RESPONSE_USER: {
+    subject: 'Offer Signed',
+    contentTop: `Congratulations on signing your offer. Details of your next steps are included in your offer letter and we are here to guide your process.`,
+    contentBottom: `  Meanwhile, click below to read our article on 5 Things Every Intending Homeowner Should Know. We look forward to an enjoyable process.`,
```

Comment:
Lets use an inline link here. Please test that this works accordingly before pushing the changes.

```suggestion
    contentBottom: `  Meanwhile, click below to read our article on <a href="www.ballers.ng">5 Things Every Intending Homeowner Should Know</a>. We look forward to an enjoyable process.`,
```